azkaban-common/src/main/java/azkaban/user/Permission.java

```diff
@@ -191,6 +191,7 @@ public int toFlags() {
     SCHEDULE(0x0000008),
     METRICS(0x0000010),
     CREATEPROJECTS(0x40000000), // Only used for roles
+    UPLOADPROJECTS(0x0008000),
```

Comment:
1. Setting lockdown.upload.projects to true will disable it for all, including admins
2. You don't have to set the above global var and default behavior is only admins and persons with permission UPLOADPROJECTS can upload.

Hope this is clear.